### ISSUE TITLE
Remote WSL stopped working with unrecognized option --port=0

### ISSUE THREAD

**Issue by Aulos** [microsoft/vscode-remote-release#328]
Issue Type: <b>Bug</b>

It stopped working after I added Remote-SSH extension, but it still doesn't work after removing it :(

Extension version: 0.35.0
VS Code version: Code - Insiders 1.34.20-insider (57b550c559b945eb9d871dbf2b2e4cb9e31e2551, 2019-05-10T17:36:45.765Z)
OS version: Windows_NT x64 10.0.17763
Fetching remote diagnostics for 'WSL' failed: VS Code Server for WSL closed unexpectedly.
Check WSL terminal for more details.

![image](https://user-images.githubusercontent.com/448931/57767878-b31e5a80-770a-11e9-9c1f-697a9473de39.png)

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|23.89GB (13.89GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

Fetching remote diagnostics for 'WSL' failed: VS Code Server for WSL closed unexpectedly.
Check WSL terminal for more details.
</details>

**Comment by hasefumi23** [comment]
I faced the same problem.

This is a workaround.

- Edit file below. (Just removing `--port=0`.)
`C:\Users\$User\.vscode-insiders\extensions\ms-vscode-remote.remote-wsl-0.35.0\scripts\wslServer.sh`
Before: `"$SHELL" -ilc "$VSCODE_REMOTE_BIN/$COMMIT/server.sh" ${INSPECT:-} --port=0 "$@"`
After: `"$SHELL" -ilc "$VSCODE_REMOTE_BIN/$COMMIT/server.sh" ${INSPECT:-} "$@"`

- Retry to launch VSCode

This may be not good, but the error has never happened.

**Comment by Aulos** [comment]
This worked, but still - it seems like a workaround

**Comment by aeschli** [comment]
duplicate of #324
